### ISSUE TITLE
Can't build ImageMagick on Centos 7 because libtiffxx.so can't be loaded

### ISSUE THREAD

**Issue by VPetukhov** [JuliaIO/ImageMagick.jl#137]
When I'm running `build ImageMagick`, it results in error:

> LoadError: LibraryProduct(nothing, ["libtiffxx"], :libtiffxx, "Prefix(/home/vp/.julia/packages/ImageMagick/at2AC/deps/usr)") is not satisfied, cannot generate deps.jl

All the other dependencies work well. Indeed, there exists file `libtiffxx.so`, but it can't be dlopened:

```julia
dlopen("/home/vp/.julia/packages/ImageMagick/at2AC/deps/usr/lib/libtiffxx.so")
```
> ERROR: could not load library "/home/vp/.julia/packages/ImageMagick/at2AC/deps/usr/lib/libtiffxx.so"
> /lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by /home/vp/.julia/packages/ImageMagick/at2AC/deps/usr/lib/libtiffxx.so)

And it's a common problem for CentOS family, as it currently doesn't support this new version of `libstdc++.so`:
```bash
nm -D /usr/lib64/libstdc++.so.6.0.19  | grep CXXABI
```
> 0000000000000000 A CXXABI_1.3
> 0000000000000000 A CXXABI_1.3.1
> 0000000000000000 A CXXABI_1.3.2
> 0000000000000000 A CXXABI_1.3.3
> 0000000000000000 A CXXABI_1.3.4
> 0000000000000000 A CXXABI_1.3.5
> 0000000000000000 A CXXABI_1.3.6
> 0000000000000000 A CXXABI_1.3.7
> 0000000000000000 A CXXABI_TM_1

So, `CXXABI_1.3.7` is the latest version. For more information see [this thread](https://github.com/displaylink-rpm/displaylink-rpm/issues/13).

Would it be possible to rebuild published `libtiffxx.so` with an older version of `libstdc++.so`?

**Comment by rsrock** [comment]
I apparently have the same issue, on a cluster that's running Scientific Linux 7.4.

Any idea how to fix this? It sounds like we're between a rock and a hard place with these libraries.

**Comment by rsrock** [comment]
Aha, just found a workaround. I provided paths to the system libtiff.so and libtiffxx.so libraries in deps/deps_tiff.jl (which I had to make by hand, using deps_jpeg.jl as a template).  They're version 4.0.6, but seem to work.

**Comment by timholy** [comment]
Should be fixed on Julia 1.3+
